src/compiler/checker.ts

```diff
@@ -28413,7 +28442,10 @@ namespace ts {
         function checkDeprecatedSignature(signature: Signature, node: CallLikeExpression) {
             if (signature.declaration && signature.declaration.flags & NodeFlags.Deprecated) {
                 const suggestionNode = getDeprecatedSuggestionNode(node);
-                errorOrSuggestion(/*isError*/ false, suggestionNode, Diagnostics._0_is_deprecated, signatureToString(signature));
+                const expr = getInvokedExpression(node);
+                const name = tryGetPropertyAccessOrIdentifierToString(expr);

```

Comment:
I would inline `expr`:
```suggestion
                const name = tryGetPropertyAccessOrIdentifierToString(getInvokedExpression(node));
```